docs/deployment.md

```diff
@@ -35,8 +35,8 @@ This is the recommended method if you provisioned the servers using the supplied
 
 [inventories/openstack-idr.py](../inventories/openstack-idr.py) is a modified version of the [default Ansible OpenStack dynamic inventory](https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/openstack.py) to make it easier to use an [Ansible SSH gateway/jump host](http://docs.ansible.com/ansible/faq.html#how-do-i-configure-a-jump-host-to-access-servers-that-i-have-no-direct-access-to), and to configure multiple networks.
 The main changes are:
-- automatic setting of SSH ProxyCommand
-- ordering of network interfaces
+- Automatic setting of SSH ProxyCommand.
+- Ordering of network interfaces.
```

Comment:
Yeah, I'd also be fine without ";": we usually follow https://en.oxforddictionaries.com/punctuation/bullet-points with our guidance being,

> Bullet point lists should begin with a capital letter and end with a full stop if each point is a complete sentence, or more than one sentence. If not, no punctuation is necessary